ion-schema/src/constraint.rs

```diff
@@ -721,7 +721,7 @@ impl OrderedElementsConstraint {
                 "ordered_elements",
                 ViolationCode::TypeMismatched,
                 &format!(
-                    "Expected {:?} of type {:?}: found {}",
+                    "Expected {} of type {:?}: found {}",
```

Comment:
Can we use display for `type_def` here?
```suggestion
                    "Expected {} of type {}: found {}",
```